WordPress/Classes/ViewRelated/Blog/SharingViewController.m

```diff
@@ -193,7 +193,7 @@ - (void)configurePublicizeCell:(UITableViewCell *)cell atIndexPath:(NSIndexPath
 
     // Check if any of the connections are broken.
     for (PublicizeConnection *pubConn in connections) {
-        if ([pubConn isBroken]) {
+        if ([pubConn isBroken] || [pubConn mustDisconnect]) {
```

Comment:
This seems to be the third different place where this boolean check is done, which, in my opinion, supports encapsulating it in the `PublicizeConnetion` object